next.dynamic.mjs

```diff
@@ -2,7 +2,9 @@
 
 import { join } from 'node:path';
 import { readFileSync } from 'node:fs';
+import { VFile } from 'vfile';
```

Comment:
as far as I can tell, we are grabbing this from transient dependencies and it should be explicitly relied upon.

```
/c/src/nodejsorg (main) 🔥 npm ls vfile
nodejs.org@1.0.0 C:\src\nodejsorg
├─┬ eslint-plugin-mdx@2.1.0
│ ├─┬ eslint-mdx@2.1.0
│ │ ├─┬ unified-engine@10.1.0
│ │ │ ├─┬ to-vfile@7.2.4
│ │ │ │ └── vfile@5.3.7 deduped
│ │ │ ├─┬ vfile-reporter@7.0.5
│ │ │ │ ├─┬ vfile-sort@3.0.1
│ │ │ │ │ └── vfile@5.3.7 deduped
│ │ │ │ └── vfile@5.3.7 deduped
│ │ │ └─┬ vfile-statistics@2.0.1
│ │ │   └── vfile@5.3.7 deduped
│ │ └── vfile@5.3.7 deduped
│ ├─┬ unified@10.1.2
│ │ └── vfile@5.3.7 deduped
│ └── vfile@5.3.7
├─┬ next-mdx-remote@4.4.1
│ ├─┬ @mdx-js/mdx@2.3.0
│ │ └── vfile@5.3.7 deduped
│ └── vfile@5.3.7 deduped
└─┬ remark-preset-lint-node@4.0.0
  ├─┬ remark-lint-checkbox-content-indent@4.1.2
  │ └─┬ vfile-location@4.1.0
  │   └── vfile@5.3.7 deduped
  ├─┬ remark-preset-lint-recommended@6.1.3
  │ └─┬ remark-lint@9.1.2
  │   └─┬ remark-message-control@7.1.1
  │     ├─┬ unified-message-control@4.0.0
  │     │ └── vfile@5.3.7 deduped
  │     └── vfile@5.3.7 deduped
  └─┬ unified-lint-rule@2.1.2
    └── vfile@5.3.7 deduped


```

Reply:
We can add it as a peerDependency. But as long as we use next-mdx-remote we should have it.

Reply:
that's a good way to introduce unexpected behavior as next-mdx-remote changes over time. I'll defer to you but always suggest explicit reliance on dependencies. this would flat out break in a pnpm environment, for example

Reply:
I’m fine eitherways. But Vfile is actually a dependency of unified itself also. And remark. So I doubt they would stop using it at any point of time as it’s part of their own ecosystem.

And I rather just mark it as a peerDependency, but I’m fine adding it as a dependency too.

Reply:
@bmuenzenmeyer I've updated the package.json